### PR TITLE
Introduce Municipal Manager Supervisor role

### PR DESCRIPTION
**Pull request by Remg**
1/
Root referents can now assign "municipal manager supervisor" role to their team members in the organizational chart

2/
3 roles can access the municipal manager attributon form:
- referents (Espace référent > Assesseurs > Communes)
- municipal chiefs (Espace Municipales 2020 > Assesseurs Communes)
- municipal manager supervisors (Espace Responsable attribution)

### CODE REVIEW

File: src/Entity/MunicipalManagerSupervisorRole.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="municipal_manager_supervisor_role")
```
Comment (ottaviano) [inline src/Entity/MunicipalManagerSupervisorRole.php]: table name seems to be the same as the class name, no need to specify here

File: templates/admin/adherent/list_status.html.twig [hunk 2]
```diff
@@ -79,6 +79,14 @@
             </span>
             <br/>
         {% endif %}
+
+        {% if object.isMunicipalManager %}
+            <span class="label label-primary" style="padding: 2px;">Responsable communale</span>
```
Comment (ottaviano) [inline templates/admin/adherent/list_status.html.twig]: `communal`

File: templates/municipal_manager_attribution/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+            {% if not form.vars.valid %}
+                <p class="text--error text--center">Le formulaire n'est pas valide</p>
+            {% endif %}
+
+            <table class="datagrid__table-manager">
+                <thead>
+                    <tr>
+                        <td class="space--0-15">Ville</td>
+                        <td>Adhérent - Responsable communale</td>
```
Comment (ottaviano) [inline templates/municipal_manager_attribution/index.html.twig]: `communal`

File: templates/municipal_manager_attribution/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+                                }) }}
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="manager__filters__section">
+                        <div class="manager__filters__group">
+                            <div class="filter__row">
+                                {{ form_row(filter_form.municipalManagerFirstName, {
+                                    label: 'Prénom du responsable communale',
```
Comment (ottaviano) [inline templates/municipal_manager_attribution/index.html.twig]: `communal`

File: templates/municipal_manager_attribution/index.html.twig [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+                                    label_attr: { class: 'filter__label' },
+                                    attr: { class: 'filter__field', placeholder: 'Saisissez un prénom' }
+                                }) }}
+                            </div>
+
+                            <div class="filter__row">
+                                {{ form_row(filter_form.municipalManagerLastName, {
+                                    label: 'Nom du responsable communale',
```
Comment (ottaviano) [inline templates/municipal_manager_attribution/index.html.twig]: `communal`

File: templates/municipal_manager_attribution/index.html.twig [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+                                }) }}
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="manager__filters__section">
+                        <div class="manager__filters__group">
+                            <div class="filter__row">
+                                {{ form_row(filter_form.municipalManagerEmail, {
+                                    label: 'Email du responsable communale',
```
Comment (ottaviano) [inline templates/municipal_manager_attribution/index.html.twig]: `communal`

File: templates/municipal_manager_supervisor/_layout.html.twig [hunk 7]
```diff
@@ -0,0 +1,56 @@
+{% extends 'base.html.twig' %}
+
+{% block page_title 'Mon espace superviseur de responsable communaux' %}
```
Comment (ottaviano) [inline templates/municipal_manager_supervisor/_layout.html.twig]: `responsables communaux`

Comment (Remg) [reply templates/municipal_manager_supervisor/_layout.html.twig]: => 'Mon espace Responsable attribution' *

**Pull request merged**